roachtest: hibernate failed

SHA: https://github.com/cockroachdb/cockroach/commits/35e138aa3c2be545fb4e17a85ea6f1b8d6525e53

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1587967&tab=artifacts#/hibernate

```
The test failed on branch=provisional_201911111508_v20.1.0-alpha.20191118, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191112-1587967/hibernate/run_1
	orm_helpers.go:174,orm_helpers.go:111,java_helpers.go:204,hibernate.go:182,hibernate.go:193,test_runner.go:697: 
		Tests run on Cockroach v20.1.0-alpha.00000000-795-g35e138a
		Tests run against hibernate 5.4.8
		7939 Total Tests Run
		7660 tests passed
		279 tests failed
		0 tests skipped
		0 tests ignored
		1 test passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList20_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/66290e232d6af34c4d09b2f90f71d520ed47dbe8

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1602090&tab=artifacts#/hibernate

```
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedHibernateProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.testmergeUninitializedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewHibernateProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testGetInitializeAssociations - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testNewProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/c128b5b9a0b1d98181bf124f775154df97948fd2

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1604949&tab=artifacts#/hibernate

```
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewHibernateProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testGetInitializeAssociations - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.testmergeUninitializedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUninitializedHibernateProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.manytomany.mapkey.ManyToManyWithMaykeyAndSchemaDefinitionTest.testRetrievingTheMapGeneratesACorrectlyQuery - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/60b7b3b2ca511cf9adb30799fcf2b8cffd6223d9

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1608628&tab=artifacts#/hibernate

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191125-1608628/hibernate/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,hibernate.go:182,hibernate.go:193,test_runner.go:697: 
		Tests run on Cockroach v20.1.0-alpha.20191118-300-g60b7b3b
		Tests run against hibernate 5.4.9
		7953 Total Tests Run
		7674 tests passed
		279 tests failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		2 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.hibernate.serialization.SessionFactorySerializationTest.testUnNamedSessionFactorySerialization - unknown (unexpected)
		--- FAIL: org.hibernate.serialization.SessionFactorySerializationTest.testNamedSessionFactorySerialization - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList20_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/9012e6449e63baca6b59c00d3e350bbc3ab0dd3b

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1611118&tab=artifacts#/hibernate

```
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUninitializedHibernateProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewHibernateProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.testmergeUninitializedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testGetInitializeAssociations - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testNewProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.manytomany.mapkey.ManyToManyWithMaykeyAndSchemaDefinitionTest.testRetrievingTheMapGeneratesACorrectlyQuery - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingProxyAssociation - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/7dec3577461a1e53ea70582de62bbd96bf512b73

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1614006&tab=artifacts#/hibernate

```
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewHibernateProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.testmergeUninitializedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.manytomany.mapkey.ManyToManyWithMaykeyAndSchemaDefinitionTest.testRetrievingTheMapGeneratesACorrectlyQuery - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testNewProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUninitializedHibernateProxy - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList2_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/70d243f36e5193bb8f0a73bf6e06a6f78c0c9f01

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1613970&tab=artifacts#/hibernate

```
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewHibernateProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.manytomany.mapkey.ManyToManyWithMaykeyAndSchemaDefinitionTest.testRetrievingTheMapGeneratesACorrectlyQuery - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUninitializedHibernateProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.testmergeUninitializedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedHibernateProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedEnhancedProxy - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testGetInitializeAssociations - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/0da08d85deeafe859345c6898e0aadd0c24236fa

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1613988&tab=artifacts#/hibernate

```
The test failed on branch=release-19.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191127-1613988/hibernate/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,hibernate.go:182,hibernate.go:193,test_runner.go:697: 
		Tests run on Cockroach v19.2.1-44-g0da08d8
		Tests run against hibernate 5.4.9
		7953 Total Tests Run
		7674 tests passed
		279 tests failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.hibernate.jpa.test.lock.LockTest.testUpdateWithPessimisticReadLockWithoutNoWait - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_2) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/39bf64d28ee5a0ab79081b8b0b29230749ea0fff

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1616610&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlDeleteNonLeaf - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testCriteriaSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.inverse.VersionedEntityWithInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithSession - unknown (unexpected)
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testQuerySuperclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedHibernateProxy - unknown (unexpected)
		--- PASS: org.hibernate.test.onetomany.inheritance.joined.MappedSuperclassMapTest.lookupEntities_entrySet - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- PASS: org.hibernate.test.onetomany.inheritance.joined.MappedSuperclassMapTest.breakReferences - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.onetomany.inheritance.joined.MappedSuperclassMapTest.lookupEntities - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testNewProxyAssociation - unknown (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlUpdateRoot - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseAsSubquery - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testStrictUnversioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingProxyAssociation - unknown (unexpected)
		--- PASS: org.hibernate.test.annotations.filter.subclass.joined.JoinedSubClassTest.testNonHumanFilter - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlDeleteLeaf - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlUpdateNonLeaf - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testUnstrictUnversioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.filter.subclass.joined.JoinedSubClassTest.testPregnantFilter - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList2_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/549d5bb865eaf9f5233cf8a068034637587b4373

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1616574&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.annotations.filter.subclass.joined.JoinedSubClassTest.testNonHumanFilter - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.BidirectionalProxyTest.testIt -  (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneSelectIgnore - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testUnstrictUnversioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.query.GroupByAliasTest.testMultiIdAlias - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testGetInitializeAssociations - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUninitializedHibernateProxy - unknown (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnManyToOne - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithJoinedTableInheritance.testBatchOrdering - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.filter.subclass.joined.JoinedSubClassTest.testPregnantFilter - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedHibernateProxy - unknown (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlDeleteLeaf - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesPositioning - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.jpa.test.criteria.basic.AggregationResultTest.testSumOfFloats - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testNewProxyAssociation - unknown (unexpected)
		--- PASS: org.hibernate.test.unionsubclass.UnionSubclassTest.testBulkOperationsWithDifferentConnections - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.inverse.VersionedEntityWithInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.filter.subclass.joined.JoinedSubClassTest.testClub - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesForward - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.subclassfilter.JoinedSubclassFilterTest.testFiltersWithJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testVersioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testQuerySubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseFailsWithFetch - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithJoinedTableInheritance.testBatchingAmongstSubClasses - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/581a2bff2f50d5626f68bc1d51af0cf05b455dfc

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1618924&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlDeleteNonLeaf - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.onetomany.OneToManyTest.testOneToManyLinkTable - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.unionsubclass.UnionSubclassTest.testBulkOperationsWithDifferentConnections - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.jpa.test.criteria.basic.AggregationResultTest.testSumOfBigDecimals - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithTablePerClassInheritance.testBatchOrdering - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testCriteriaSuperclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseAsSubqueryWithKeyAndOtherJoinReference - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithJoinedTableInheritance.testBatchOrdering - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.BidirectionalProxyTest.testIt - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bulkid.GlobalTemporaryTableBulkCompositeIdTest.testUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesSingleRowResultSet - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.jpa.test.criteria.basic.AggregationResultTest.testSumOfDoubles - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.noninverse.EntityWithNonInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesPositioning - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testStrictUnversioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneSelectException - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseFailsWithFetch - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testCollectionFetchWithDistinctionAndLimit - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testGetSuperclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.jpa.test.criteria.basic.AggregationResultTest.testSumOfBigIntegers - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithSelectListUsingJoins - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.onetomany.inheritance.joined.MappedSuperclassMapTest.lookupEntities_entrySet - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.inverse.EntityWithInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.query.GroupByAliasTest.testCompoundIdAlias - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList20_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/d3574ad671bd3631e780510235485681720c2b8f

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1622074&tab=artifacts#/hibernate

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191203-1622074/hibernate/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,hibernate.go:173,hibernate.go:184,test_runner.go:697: 
		Tests run on Cockroach v20.1.0-alpha.20191118-443-gd3574ad
		Tests run against hibernate HHH-13724-cockroachdb-support
		7867 Total Tests Run
		7713 tests passed
		154 tests failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.hibernate.test.subclassfilter.JoinedSubclassFilterTest.testFiltersWithJoinedSubclass - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList20_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/281b3f9ac2500c882dfcd5d223e8cc23ba35b3ea

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1622092&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.inverse.VersionedEntityWithInverseOneToManyJoinFailureExpectedTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatelessTest.testExistingInitializedAssociationLeafSubclass - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlUpdateRoot - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlDeleteLeaf - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.inverse.VersionedEntityWithInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingInitializedAssociationLeafSubclass - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingProxyAssociation - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.onetomany.inheritance.joined.MappedSuperclassMapTest.breakReferences - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.query.GroupByAliasTest.testMultiIdAlias - unknown (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testGetInitializeAssociations - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.jpa.test.criteria.basic.AggregationResultTest.testSumOfLongs - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.quote.QuoteTest.testUnionSubclassEntityQuoting - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewHibernateProxyAssociation - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithJoinedTableInheritance.testBatchOrdering - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlDeleteNonLeaf - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.BidirectionalProxyTest.testIt - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testUnstrictUnversioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.joinedSubclass.JoinedSubclassBulkManipTest.testHqlDeleteOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.query.GroupByAliasTest.testCompoundIdAlias - unknown (unexpected)
		--- PASS: org.hibernate.test.unionsubclass.UnionSubclassTest.testBulkOperationsWithDifferentConnections - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_2) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/20376489afba84e533d8511f4c9a59b15eec1371

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1624121&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.quote.QuoteTest.testUnionSubclassEntityQuoting - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.BidirectionalProxyTest.testIt - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.noninverse.VersionedEntityWithNonInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.onetoone.hhh9798.OneToOneJoinTableTest.storeNonUniqueRelationship - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testUnstrictUnversioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlDeleteRoot - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.LazyInitializationWithoutInlineDirtyTrackingTest.test - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testAbstractClassAssociation - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.mixed.MixedTest.testMixedInheritance - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.noninverse.EntityWithNonInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.lob.VersionedLobTest.testVersionUnchangedByteArray - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.annotations.lob.LobTest.testBlob - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadNonOwningOneToOne - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.filter.subclass.joined.JoinedSubClassTest.testIqFilter - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.LazyBasicFieldMergeTest.test - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.hql.joinedSubclass.JoinedSubclassBulkManipTest.testHqlUpdateOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneSelectException - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.onetomany.inheritance.joined.MappedSuperclassMapTest.lookupEntities_entrySet - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.jpa.test.lob.BlobTest.testBlobSerialization - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.hql.joinedSubclass.JoinedSubclassBulkManipTest.testHqlDeleteOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.jpa.test.query.ConstructorResultNativeQueryTest.testMultipleConstructorResultNativeQuery - unknown (unexpected)
		--- PASS: org.hibernate.test.onetomany.inheritance.joined.MappedSuperclassMapTest.breakReferences - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testVersioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_2) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/ed717cbaf741e3a32c76db25b16a59dc2a8221d7

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1624103&tab=artifacts#/hibernate

```
		--- FAIL: org.hibernate.test.annotations.lob.VersionedLobTest.testVersionUnchangedPrimitiveCharArray - unknown (unexpected)
		--- FAIL: org.hibernate.test.annotations.lob.VersionedLobTest.testClob - unknown (unexpected)
		--- FAIL: org.hibernate.test.annotations.lob.LobTest.testClob - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testRandomAccess - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testQueryAndDeleteDEntity - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- FAIL: org.hibernate.test.subclassfilter.JoinedSubclassFilterTest.testFiltersWithJoinedSubclass - unknown (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLazyAssociationSameAsNonLazyInPC - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.LazyBasicFieldMergeTest.test - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testExistingInitializedAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.annotations.lob.VersionedLobTest.testVersionUnchangedString - unknown (unexpected)
		--- FAIL: org.hibernate.test.type.LobUnfetchedPropertyTest.testClob - unknown (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testAbstractClassAssociation - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadOwningOneToOne - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedEnhancedProxy - unknown (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testNullManyToOneHql - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testQueryAndDeleteEEntity - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesNoProxyFactoryWithSubclassesStatefulTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- PASS: org.hibernate.test.mixed.MixedTest.testMixedInheritance - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testFetchingScroll - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewEnhancedProxyAssociation - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingHibernateProxyAssociationLeafSubclass - unknown (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.LazyInitializationWithoutInlineDirtyTrackingTest.test - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - unknown (unexpected)
		--- FAIL: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedHibernateProxy - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList20_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/9ad9eb5fb8806e4b74546910ca8bda66786d4288

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1626352&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.LazyInitializationWithoutInlineDirtyTrackingTest.test - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testRandomAccess - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.jpa.test.criteria.basic.AggregationResultTest.testSumOfBigIntegers - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.id.hhh12973.SequenceMismatchStrategyDefaultExceptionTest.test - unknown (unexpected)
		--- PASS: org.hibernate.jpa.test.criteria.basic.AggregationResultTest.testSumOfIntegers - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.inverse.VersionedEntityWithInverseOneToManyJoinFailureExpectedTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.jpa.test.criteria.basic.AggregationResultTest.testSumOfLongs - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.schemavalidation.LongVarcharValidationTest.testValidation[1] - https://github.com/cockroachdb/cockroach/issues/16769 (unexpected)
		--- PASS: org.hibernate.test.schemavalidation.NumericValidationTest.testValidation[0] - https://github.com/cockroachdb/cockroach/issues/16769 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testSimpleInsertWithNamedParam - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteOnDiscriminatorSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneSelectIgnore -  (unexpected)
		--- PASS: org.hibernate.test.hql.joinedSubclass.JoinedSubclassBulkManipTest.testHqlDeleteOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.onetomany.OneToManyTest.testOneToManyLinkTable - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.filter.subclass.joined.JoinedSubClassTest.testClub - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithJoinedTableInheritance.testBatchingAmongstSubClasses - https://github.com/cockroachdb/cockroach/issues/26738 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testSimpleSelect - pass? (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertAcrossMappedJoinFails - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.tm.CMTTest.testConcurrentCachedQueries - pass? (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneJoinException -  (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithMismatchedTypes - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseAsSubqueryWithKey - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testVersioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.dialect.functional.SequenceInformationTest.test - unknown (unexpected)
		--- FAIL: org.hibernate.test.typeoverride.TypeOverrideTest.testStandardBasicSqlTypeDescriptor - unknown (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneSelectException -  (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList2_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/c5647e9ab0282e182805a9d1a44259790bea8558

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1626316&tab=artifacts#/hibernate

```
		--- FAIL: org.hibernate.query.GroupByAliasTest.testMultiIdAlias - unknown (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testStrictUnversioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesForward - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnComponent - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnDiscriminatorSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.jpa.compliance.tck2_2.caching.CachingWithSecondaryTablesTests.testVersioned - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testQueryAndDeleteEEntity - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.LazyBasicFieldMergeTest.test - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.ProxyDeletionTest.testGetAndDeleteEEntity -  (unexpected)
		--- PASS: org.hibernate.test.hql.joinedSubclass.JoinedSubclassBulkManipTest.testHqlDeleteOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithMismatchedTypes - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseFailsWithFetch - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnMammal - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteOnMappedJoin - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateSetNullOnDiscriminatorSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testEntityAndOneToOneReturnedByQuery - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadNonOwningOneToOne - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneSelectIgnore - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.inverse.VersionedEntityWithInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testGetAndDeleteEEntity - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testCriteriaSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.query.GroupByAliasTest.testSingleIdAlias - unknown (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneJoinIgnore - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testWhere - pass? (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testAbstractClassAssociation - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.mixed.MixedTest.testMixedInheritance - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/c50e3f43a5ccb3ed2700af002a25c10f59e877a5

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1626334&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnMammal - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClause - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.ProxyDeletionTest.testGetAndDeleteEEntity - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testFetchingScroll - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithTablePerClassInheritance.testBatchingAmongstSubClasses - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadNonOwningOneToOne - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseAsSubqueryWithEqualOperator - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlUpdateRoot - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneJoinIgnore - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.jpa.test.query.ConstructorResultNativeQueryTest.testMultipleConstructorResultNativeQuery - unknown (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.inverse.VersionedEntityWithInverseOneToManyJoinFailureExpectedTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneSelectIgnore - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlUpdateNonLeaf - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseFailsWithFetch - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithSelectListUsingJoins - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.notfound.OptionalEagerMappedByNotFoundTest.testOneToOneSelectException - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.testmergeUninitializedEnhancedProxy - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingProxyAssociation - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseAsSubqueryWithKey - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.loadplans.process.inheritance.Test.basicTest - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testWhere -  (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesReverse - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.BidirectionalProxyTest.testIt - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUninitializedHibernateProxy - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_2) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/e1c1a9ae765730782f3adc2298a76d6a07ad055d

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1626298&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteUnionSubclassAbstractRoot - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesPositioning - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseFailsWithFetch - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadNonOwningOneToOne - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnAnimal - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadAndDeleteEEntity - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.hql.CriteriaHQLAlignmentTest.testCountReturnValues - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseAsSubqueryWithKey - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.jpa.test.query.ConstructorResultNativeQueryTest.testConstructorResultNativeQuerySpecifyingType - unknown (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyTest.testScrollableWithStatelessSession - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertAcrossMappedJoinFails - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.LazyInitializationWithoutInlineDirtyTrackingTest.test - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- FAIL: org.hibernate.jpa.test.query.ConstructorResultNativeQueryTest.testConstructorResultNativeQuery - unknown (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testTempTableGenerationIsolation - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithJoinedTableMultiLevelInheritance.testBatchingAmongstSubClasses - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testWhere -  (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesSingleRowResultSet - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithSession - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithStatelessSession - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.QueryScrollingWithInheritanceTest.testScrollableWithSession - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateWithWhereExistsSubquery - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithManyToOne - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList20_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/28f216e1bd53da872a759a98779144c7f70f33a3

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1629573&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithSubqueriesAndNamedParams - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateSetNullUnionSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseAsSubqueryWithKey - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testSimpleInsert - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteWithMetadataWhereFragments - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadNonOwningOneToOne - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testNullManyToOneHql - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadOwningOneToOne - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateSetNullOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithManyToOne - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hqlfetchscroll.QueryScrollingWithInheritanceTest.testScrollableWithSession - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- FAIL: org.hibernate.test.typeoverride.TypeOverrideTest.testStandardBasicSqlTypeDescriptor - unknown (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClause - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesSingleRowResultSet - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithStatelessSession - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnMammal - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testComponentNullnessChecks - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnAnimal - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteOnDiscriminatorSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.converter.AttributeConverterTest.testBasicConverterApplication - unknown (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testNonAbstractAssociationWithSubclassValue - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteUnionSubclassConcreteSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.jpa.test.query.ConstructorResultNativeQueryTest.testConstructorResultNativeQuery - unknown (unexpected)
		--- PASS: org.hibernate.test.where.hbm.LazyManyToManyNonUniqueIdNotFoundWhereTest.testInitializeFromNonUniqueAssociationTable - https://github.com/cockroachdb/cockroach/issues/12123 (unexpected)
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesPositioning - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList20_1) is available in the artifacts' hibernate log

```

SHA: https://github.com/cockroachdb/cockroach/commits/5224df9c8bd9a36dfde24ae3abe7b7e42a5d9660

Parameters:

To repro, try:

```
# Don't forget to check out a clean suitable branch and experiment with the
# stress invocation until the desired results present themselves. For example,
# using stress instead of stressrace and passing the '-p' stressflag which
# controls concurrency.
./scripts/gceworker.sh start && ./scripts/gceworker.sh mosh
cd ~/go/src/github.com/cockroachdb/cockroach && \
stdbuf -oL -eL \
make stressrace TESTS=hibernate PKG=roachtest TESTTIMEOUT=5m STRESSFLAGS='-maxtime 20m -timeout 10m' 2>&1 | tee /tmp/stress.log
```

Failed test: https://teamcity.cockroachdb.com/viewLog.html?buildId=1631737&tab=artifacts#/hibernate

```
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testQuerySuperclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesTest.testExistingEnhancedProxyAssociationLeafSubclassOnly - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.converter.lob.ConverterAndLobTest.basicTest - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnComponent - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testComponentNullnessChecks - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.where.annotations.LazyManyToManyNonUniqueIdWhereTest.testInitializeFromUniqueAssociationTable - https://github.com/cockroachdb/cockroach/issues/12123 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testNewHibernateProxyAssociation - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.optlock.OptimisticLockTest.testOptimisticLockDirtyDelete - Expected Failure (unexpected)
		--- PASS: org.hibernate.test.hql.UpdateJoinedSubclassCorrelationTest.testJoinedSubclassUpdateWithCorrelation - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.joinedSubclass.JoinedSubclassBulkManipTest.testHqlDeleteOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.QueryScrollingWithInheritanceProxyEagerManyToOneTest.testScrollableWithStatelessSession - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateSetNullUnionSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyWithSubclassesStatelessTest.testExistingHibernateProxyAssociationLeafSubclass - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateSetNullOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedEnhancedProxy - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- PASS: org.hibernate.test.where.hbm.LazyManyToManyNonUniqueIdNotFoundWhereTest.testInitializeFromUniqueAssociationTable - https://github.com/cockroachdb/cockroach/issues/12123 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.LazyInitializationWithoutInlineDirtyTrackingTest.test - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testNullManyToOneHql - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.LazyToOnesProxyMergeWithSubclassesTest.mergeUpdatedHibernateProxy - https://github.com/cockroachdb/cockroach/issues/26508 (unexpected)
		--- FAIL: org.hibernate.query.GroupByAliasTest.testCompoundIdAlias - unknown (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertIntoSuperclassPropertiesFails - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.where.annotations.LazyOneToManyNonUniqueIdWhereTest.testInitializeFromNonUniqueAssociationTable - https://github.com/cockroachdb/cockroach/issues/12123 (unexpected)
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testCriteriaSuperclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_2) is available in the artifacts' hibernate log

```

[(roachtest).hibernate failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634692&tab=buildLog) on [provisional_201912092102_v19.2.2@c85d435c2caa7a3066b1affdcc34b7ed6a6f3653](https://github.com/cockroachdb/cockroach/commits/c85d435c2caa7a3066b1affdcc34b7ed6a6f3653):

```
The test failed on branch=provisional_201912092102_v19.2.2, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191209-1634692/hibernate/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,hibernate.go:173,hibernate.go:184,test_runner.go:700: 
		Tests run on Cockroach v19.2.1-87-gc85d435
		Tests run against hibernate HHH-13724-cockroachdb-support
		7818 Total Tests Run
		7814 tests passed
		4 tests failed
		0 tests skipped
		0 tests ignored
		2 tests passed unexpectedly
		4 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- PASS: org.hibernate.query.GroupByAliasTest.testMultiIdAlias - failing on Postgres (unexpected)
		--- FAIL: org.hibernate.jpa.test.criteria.QueryBuilderTest.testDateTimeFunctions - unknown (unexpected)
		--- PASS: org.hibernate.query.GroupByAliasTest.testCompoundIdAlias - failing on Postgres (unexpected)
		--- FAIL: org.hibernate.test.legacy.ParentChildTest.testLoadAfterNonExists - unknown (unexpected)
		--- FAIL: org.hibernate.test.hql.ASTParserLoadingTest.testStandardFunctions - unknown (unexpected)
		--- FAIL: org.hibernate.jpa.test.query.NativeQueryOrdinalParametersTest.testCteNativeQueryOrdinalParameter - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_2) is available in the artifacts' hibernate log

```

<details><summary>details</summary>
<p>


Artifacts: [/hibernate](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634692&tab=artifacts#/hibernate)
```
make stressrace TESTS=hibernate PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(roachtest).hibernate failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634845&tab=buildLog) on [provisional_201912100444_v2.1.10@9ad9eb5fb8806e4b74546910ca8bda66786d4288](https://github.com/cockroachdb/cockroach/commits/9ad9eb5fb8806e4b74546910ca8bda66786d4288):

```
		--- PASS: org.hibernate.test.hql.ScrollableCollectionFetchingTest.testScrollingJoinFetchesReverse - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithMismatchedTypes - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseFailsWithFetch - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.immutable.entitywithmutablecollection.noninverse.EntityWithNonInverseOneToManyJoinTest.testOneToManyCollectionOptimisticLockingWithUpdate - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.lob.LobStringTest.testHqlQuery - https://github.com/cockroachdb/cockroach/issues/26725 (unexpected)
		--- PASS: org.hibernate.test.quote.QuoteTest.testUnionSubclassEntityQuoting - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testMultipleRefsToSameParam - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.insertordering.InsertOrderingWithTablePerClassInheritance.testBatchOrdering - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.filter.subclass.joined.JoinedSubClassTest.testClub - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testUpdateOnComponent - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.onetomany.OneToManyTest.testOneToManyLinkTable - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithSubqueriesAndNamedParams - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.BidirectionalProxyTest.testIt -  (unexpected)
		--- PASS: org.hibernate.test.hql.joinedSubclass.JoinedSubclassBulkManipTest.testHqlUpdateOnJoinedSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClause - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testSimpleInsert - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.where.hbm.LazyManyToManyNonUniqueIdWhereTest.testInitializeFromNonUniqueAssociationTable - https://github.com/cockroachdb/cockroach/issues/12123 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadAndDeleteDEntity - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testLoadAndDeleteEEntity - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.annotations.onetoone.hhh9798.OneToOneJoinTableTest.storeNonUniqueRelationship - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteRestrictedOnManyToOne - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.bytecode.enhancement.lazy.proxy.FetchGraphTest.testNullManyToOneHql - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.filter.hql.JoinedFilteredBulkManipulationTest.testFilteredJoinedSubclassHqlUpdateRoot - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testDeleteUnionSubclassLeafSubclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testSimpleNativeSQLInsert - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.join.SubclassesWithSamePropertyNameTest.testGetSuperclass - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.WithClauseTest.testWithClauseAsSubquery - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- PASS: org.hibernate.test.hql.ASTParserLoadingTest.testCollectionFetchWithDistinctionAndLimit - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		--- FAIL: org.hibernate.test.dialect.functional.SequenceInformationTest.test - unknown (unexpected)
		--- PASS: org.hibernate.test.hql.BulkManipulationTest.testInsertWithSelectListUsingJoins - https://github.com/cockroachdb/cockroach/issues/5807 (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList2_1) is available in the artifacts' hibernate log

```

<details><summary>details</summary>
<p>


Artifacts: [/hibernate](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634845&tab=artifacts#/hibernate)
```
make stressrace TESTS=hibernate PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(roachtest).hibernate failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634885&tab=buildLog) on [release-19.1@6e95a8334dea98fdfa93fdc2e981935947089e32](https://github.com/cockroachdb/cockroach/commits/6e95a8334dea98fdfa93fdc2e981935947089e32):

```
The test failed on branch=release-19.1, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191210-1634885/hibernate/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,hibernate.go:173,hibernate.go:184,test_runner.go:700: 
		Tests run on Cockroach v19.1.5-78-g6e95a83
		Tests run against hibernate HHH-13724-cockroachdb-support
		7818 Total Tests Run
		7724 tests passed
		94 tests failed
		0 tests skipped
		0 tests ignored
		0 tests passed unexpectedly
		4 tests failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.hibernate.jpa.test.query.NativeQueryOrdinalParametersTest.testCteNativeQueryOrdinalParameter - unknown (unexpected)
		--- FAIL: org.hibernate.jpa.test.criteria.QueryBuilderTest.testDateTimeFunctions - unknown (unexpected)
		--- FAIL: org.hibernate.test.hql.ASTParserLoadingTest.testStandardFunctions - unknown (unexpected)
		--- FAIL: org.hibernate.test.legacy.ParentChildTest.testLoadAfterNonExists - unknown (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList19_1) is available in the artifacts' hibernate log

```

<details><summary>details</summary>
<p>


Artifacts: [/hibernate](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634885&tab=artifacts#/hibernate)
```
make stressrace TESTS=hibernate PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

[(roachtest).hibernate failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634867&tab=buildLog) on [master@734e357412dadafcd6084b8eab8e251e44e86b4a](https://github.com/cockroachdb/cockroach/commits/734e357412dadafcd6084b8eab8e251e44e86b4a):

```
The test failed on branch=master, cloud=gce:
test artifacts and logs in: /home/agent/work/.go/src/github.com/cockroachdb/cockroach/artifacts/20191210-1634867/hibernate/run_1
	orm_helpers.go:181,orm_helpers.go:111,java_helpers.go:204,hibernate.go:173,hibernate.go:184,test_runner.go:700: 
		Tests run on Cockroach v20.1.0-alpha.20191118-591-g734e357
		Tests run against hibernate HHH-13724-cockroachdb-support
		7818 Total Tests Run
		7817 tests passed
		1 test failed
		0 tests skipped
		0 tests ignored
		2 tests passed unexpectedly
		1 test failed unexpectedly
		0 tests expected failed but skipped
		0 tests expected failed but not run
		---
		--- FAIL: org.hibernate.test.legacy.ParentChildTest.testLoadAfterNonExists - unknown (unexpected)
		--- PASS: org.hibernate.query.GroupByAliasTest.testCompoundIdAlias - failing on Postgres (unexpected)
		--- PASS: org.hibernate.query.GroupByAliasTest.testMultiIdAlias - failing on Postgres (unexpected)
		For a full summary look at the hibernate artifacts 
		
		An updated blacklist (hibernateBlackList20_1) is available in the artifacts' hibernate log

```

<details><summary>details</summary>
<p>


Artifacts: [/hibernate](https://teamcity.cockroachdb.com/viewLog.html?buildId=1634867&tab=artifacts#/hibernate)
```
make stressrace TESTS=hibernate PKG=./pkg/roachtest TESTTIMEOUT=5m STRESSFLAGS=-timeout 5m' 2>&1
```

<sub>powered by [pkg/cmd/internal/issues](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)</sub>
</p>
</details>

--- a/pkg/cmd/roachtest/hibernate_blacklist.go
+++ b/pkg/cmd/roachtest/hibernate_blacklist.go
@@ -22,15 +22,9 @@ var hibernateBlacklists = blacklistsForVersion{
 // Please keep these lists alphabetized for easy diffing.
 // After a failed run, an updated version of this blacklist should be available
 // in the test log.
-var hibernateBlackList20_1 = blacklist{
-	"org.hibernate.query.GroupByAliasTest.testCompoundIdAlias": "failing on Postgres",
-	"org.hibernate.query.GroupByAliasTest.testMultiIdAlias":    "failing on Postgres",
-}
+var hibernateBlackList20_1 = blacklist{}
 
-var hibernateBlackList19_2 = blacklist{
-	"org.hibernate.query.GroupByAliasTest.testCompoundIdAlias": "failing on Postgres",
-	"org.hibernate.query.GroupByAliasTest.testMultiIdAlias":    "failing on Postgres",
-}
+var hibernateBlackList19_2 = blacklist{}
 
 var hibernateBlackList19_1 = blacklist{
 	"org.hibernate.jpa.test.indetifier.AssignedInitialValueTableGeneratorConfiguredTest.testTheFirstGeneratedIdIsEqualToTableGeneratorInitialValuePlusOne": "6583",

--- a/pkg/cmd/roachtest/psycopg_blacklist.go
+++ b/pkg/cmd/roachtest/psycopg_blacklist.go
@@ -275,7 +275,6 @@ var psycopgBlackList20_1 = blacklist{
 	"tests.test_transaction.TransactionTests.test_rollback":                                                  "5807",
 	"tests.test_types_basic.TypesBasicTests.testArray":                                                       "32552",
 	"tests.test_types_basic.TypesBasicTests.testArrayOfNulls":                                                "32552",
-	"tests.test_types_basic.TypesBasicTests.testEmptyArray":                                                  "23299",
 	"tests.test_types_basic.TypesBasicTests.testEmptyArrayRegression":                                        "36179",
 	"tests.test_types_basic.TypesBasicTests.testNestedArrays":                                                "32552",
 	"tests.test_types_basic.TypesBasicTests.testNestedEmptyArray":                                            "32552",

